tests/api/v1/endpoints/test_html_sanitization.py

```diff
@@ -0,0 +1,13 @@
+from src.fides.api.custom_types import validate_html_str
+
+
+def test_html_sanitization():
+    """Test that the validate_html_str function sanitizes the HTML string correctly, keeping the target attribute"""
+
+    test_html = 'See our <a href="https://www.somerandomwebsite.com/cookies/" target="_blank">privacy policy</a>'
+
+    sanitized = validate_html_str(test_html)
+
+    expected = 'See our <a href="https://www.somerandomwebsite.com/cookies/" target="_blank" rel="noopener noreferrer">privacy policy</a>'
+
+    assert sanitized == expected
```

Comment:
Hi @Vagoasdf, if we don't have any test cases in the codebase already that check if malicious html is being sanitized correctly, let's add them.

A few examples:

1. Script injection
```html
See our <a href="https://example.com" target="_blank"><script>alert("xss")</script>privacy policy</a> here.
```

2. Event handler injection
```html
See our <a href="https://example.com" target="_blank" onclick="alert(1)">privacy policy</a> here.
```

3. JavaScript URL injection
```html
See our <a href="javascript:alert(1)" target="_blank">privacy policy</a> here.
```

Reply:
Adding them! 🆗